includes/polling/ports.inc.php

```diff
@@ -175,6 +175,10 @@
     'dot3StatsDuplexStatus',
 ];
 
+$pvid_oids = [
```

Comment:
I feel like this should be inlined it is just one oid and I don't expect more to be added.